packages/@aws-cdk/aws-eks/lib/kubectl-handler/helm/__init__.py

```diff
@@ -24,7 +41,7 @@ def helm_handler(event, context):
     release      = props['Release']
     chart        = props['Chart']
     version      = props.get('Version', None)
-    wait         = props.get('Wait', False)
+    wait         = decode_boolean(props.get('Wait', False))
```

Comment:
Why is this needed? was this a bug??

Reply:
yes, it was

Reply:
I guess the bug was when it was explicitly set to “false” and evaluated as truthy. If it was not set it evaluates as False, which is fine and the default behavior, and If “Wait” was “true” it still evaluates as truthy. 

If that’s the case, I rather just ensure that Wait is assigned an empty string or undefined from the construct when it is false instead of playing with decoding.

Reply:
alright